ElementX/Sources/Services/Room/RoomMember/RoomMemberProxy.swift

```diff
@@ -0,0 +1,54 @@
+//
+// Copyright 2023 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import MatrixRustSDK
+
+final class RoomMemberProxy: RoomMemberProxyProtocol {
+    private let member: RoomMemberProtocol
+
+    init(with member: RoomMemberProtocol) {
```

Comment:
Again, only if you want to handle the changes elsewhere, we could rename this one too. I think `init`s should be clear about what is being passed in.
```suggestion
    init(member: RoomMemberProtocol) {
```